handlers/service.js

```diff
@@ -11,6 +11,36 @@ const lineEnd = config.get('withoutSemi') ? '' : ';'
 const template = handlebars.compile(fs.readFileSync(path.join(__dirname, '..', 'templates', 'ServiceClass.js.hbs'), 'utf8'))
 
 
+const genServiceClass = (service, s, proto) => {
+  const members = []
+
+  service.methodList.forEach((rpc, r) => {
+    const paramComments = [`     * @param {${baseNamespace}${rpc.inputType}} payload`]
```

Comment:
This whole file diff locks messed up and seems to have a lot of duplicated code. And at least this line still has the wrong order of parameterName + type

Reply:
I'm glad you're paying attention! Ya, something weird must have happened when I synced my fork. Should be good now.